### DISCUSSION TITLE
How to read/delete SMS from outbox?

### DISCUSSION THREAD

**Discussion by krzysiekhelp** [Salamek/huawei-lte-api#177] [General]
How to read and delete SMS from outbox? 

BoxTypeEnum.LOCAL_OUTBOX - doesn't work

**Comment by krzysiekhelp** [comment]
I found BoxTypeEnum.LOCAL_SENT

**Comment by Salamek** [comment]
`BoxTypeEnum.LOCAL_SENT` works for me...
